data/weather-symbol-modes/wet_categories.json

```diff
@@ -0,0 +1,5 @@
+{
+    "extreme_convection": [30, 29, 28, 21, 20, 19],
+    "frozen": [27, 26, 25, 24, 23, 22, 18, 17, 16],
+    "liquid": [15, 14, 13, 12, 11, 10, 9]
```

Comment:
I don't think we need the nighttime symbols in these lists, as the night -> day symbol unification is done first in the summary symbol calculation.

Reply:
I've removed the night-time codes.